manywheel/Dockerfile

```diff
@@ -62,10 +63,6 @@ ADD ./common/install_patchelf.sh install_patchelf.sh
 RUN bash ./install_patchelf.sh && rm install_patchelf.sh
 RUN cp $(which patchelf) /patchelf
 
-FROM base as breakpad
-ADD ./common/install_breakpad.sh install_breakpad.sh
-RUN bash ./install_breakpad.sh
```

Comment:
This fails because it pins a commit that has been rebased out of the repo's history. Deleted because `breakpad` is now a git submodule in pytorch, since pytorch/pytorch#63186